test/screenshot/mdc-fab/fixture.scss

```diff
@@ -22,3 +22,7 @@
   width: 81px;
   height: 81px;
 }
+
+.test-cell--extended {
+  width: 191px;
```

Comment:
Also add a height (also ending in 1)

Don't forget to regenerate goldens (it looks like the ones currently in this PR aren't current with this width update)